bool status
---
string object
string color
